diagnostics/physics/mld_eval.py

```diff
@@ -49,18 +59,28 @@ def plot_mld_eval(pp_root):
 
     # the two observed datasets have the same grid
     delta_deboyer = mom_rg - deboyer_mld_winter
+    logger.info("DEBOYER_MLD_WINTER: %s",deboyer_mld_winter)
+    logger.info("DELTA_DEBOYER: %s",delta_deboyer)
+
+    # Set projection of each grid in the plot
+    # For now, sst_eval.py will only support a projection for the arctic and a projection for all other domains
+    if config['projection_grid'] == 'NorthPolarStereo':
```

Comment:
As Andrew suggested, we could add the `central_longitude` parameter here. We could even allow users to choose their preferred value in the YAML configuration.